client/js/components/month/month.js

```diff
@@ -1,15 +1,9 @@
 /** @jsx React.DOM */
 var React = require('react');
 var MonthEventsContainer = require('./month-events-container.js');
-var MonthStore = require('../../stores/month-store.js');
-var MonthHeader = require('./month-header.js');
-var MonthData = require('../../monthData.js');
-
-function monthData () {
-  return {
-    gigs: MonthStore.getMonthData()
-  };
-}
+var MonthStore = require('../../stores/month-store');
+var MonthHeader = require('./month-header');
+var MonthData = require('../../monthData');
```

Comment:
Could catch, Ill submit another PR for the initilization now that flux has been implemented for the month componenet